packages/plugins/videos-ui/src/lib/+state/video.action.ts

```
@@ -0,0 +1,38 @@
+		url
+	}));
+
+	public static retryDownload = createAction('[Videos] Retry Download', (url: string) => ({
+		url
+	}));
+
+	public static fetchVideosAndExclude = createAction(
+		'[Videos] Fetch Videos and Exclude',
+		(id: string, params?: any) => ({ id, params })
+	);
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)